### ISSUE TITLE
How to use ajv-formats in JS ?

### ISSUE THREAD

**Issue by sabrineChe** [ajv-validator/ajv#1582]
I'm using the minified version of ajv7 in Javascript, and i want to support other formats, which are in the ajv-formats package.
I want to import ajv-formats, which is written in typescript, to my Javascript project.
Is there a possibility to make it ? Does a minified version of ajv-formats exist ?

**Comment by epoberezkin** [comment]
No, it is not provided. There is a compiled JS code though that it would be much more efficient to bundle with your code rather than use as a separate bundle as it's really small.
